### ISSUE TITLE
Make it easier to donate to YASB 2 project

### ISSUE THREAD

**Issue by elderling** [raithos/xwing#914]
### Expected Behavior

I expected it to be easy to donate in light of the major 2.5 changes.

### Actual Behavior

Link to donate is buried under "About" menu.

### Language

Presumably all but I only "tested" English.

**Comment by Patschke** [comment]
Well, there is no organization within the maintainers of YASB, therefore donating is not that easy. Georandr (original author of 1.0 YASB) is still running the current backend server (where your squads are stored), but does not contribute to the code anymore. Raithos and I actually work on the current YASB code, adding features and fixing bugs as required. Raithos did add a personal paypal link in the About menu, I didn't do that so far. For me it would be preferred if people contributed by adding code (even non programmers can do so. E.g. adding the QuickBuilds which I maintained so far, isn't hard - it's just time consuming), so I kinda don't really want to add too much incentive of having people expect that I'll do certain work (which they will expect if they donate money), just because I did it the last few years.
